devnet.yaml

```diff
@@ -4,42 +4,48 @@
 # start a devnet with any local changes
 
 deployments:
-  - type: EigenLayer
-    # This commit corresponds to the incredible-squaring commit below
-    ref: "v0.3.3-mainnet-rewards"
-    strategies: [MockETH]
-    operators:
-      - name: operator1
-        keys: operator1_ecdsa_keys
-        strategies:
-          # strategy_name: number_of_tokens
-          MockETH: 100000000000000000
-      - name: operator2
-        keys: operator2_ecdsa_keys
-        strategies:
-          # strategy_name: number_of_tokens
-          MockETH: 100000000000000000
-      - name: operator3
-        keys: operator3_ecdsa_keys
-        strategies:
-          # strategy_name: number_of_tokens
-          MockETH: 100000000000000000
+
+  - name: contract_registry
+    repo: "."
+    contracts_path: "contracts"
+    script: script/DeployContractsRegistry.s.sol
+    env:
+      PRIVATE_KEY: "{{.keys.deployer_key.private_key}}"
```

Comment:
Do we need to deploy the contracts registry?

Reply:
We don't use any contract from registry rn . If i remove this:

Addresses will be inconsistent with IS-rs version
Should i do it?

Reply:
removed 👍